docker-compose.example.yml

```diff
@@ -0,0 +1,68 @@
+version: "3"
+services:
+  # The postgres server
+  database:
+    # Pulling the image in the latest tag is a disaster.
+    image: postgres:13-alpine
+    environment: 
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRE_DB=postgres
+    volumes:
+    - dbdata:/var/lib/postgresql/data
+  
+  # https://github.com/benawad/dogehouse/issues/101#issuecomment-778605325
+  adminer:
+    image: adminer
+    restart: "no"
+    ports:
+      - 5051:8080
+    depends_on:
+      - database
+  
+  # The WSS server used to establish and manage voice and text rooms
+  voice_server:
+    build:
+      context: ./shawarma
+      dockerfile: Dockerfile
+    depends_on: 
+      - backend
+
+  # The backend server
+  backend:
+    build: 
+      context: ./kousa
+      dockerfile: Dockerfile
+    depends_on: 
+      - database
+    environment:
+      # Database of the Postgres server
+      # Should be same as your Postgres DB config above
+      - DATABASE_URL=ecto://postgres:postgres@127.0.0.1/postgres
+      
+      # GitHub username or user ID for the instance owner
+      # By default, this should be either Ben's or yours.
+      - BEN_GITHUB_ID={username}
+      
+      # The React frontend
+      # Leave it as is if doing development locally
+      # Otherwise, edit this.
+      - RABBITMQ_URL=amqp://192.168.1.165:3000
+      
+      # Not needed in dev instances.
+      # But it up to you if you want this.
+      # - SENTRY_DSN=NO_IDEA_WHAT_THIS_IS
+      
+      # OAuth stuff
+      - GITHUB_CLIENT_ID=5d5xxxxxxab70
+      - GITHUB_CLIENT_SECRET=48xxxxxxxxxxxxd1cd5
+      
+      # Enter some random stuff in these variables
+      # to generate some spice.
+      - ACCESS_TOKEN_SECRET=8f51exxxxxx7211dd
+      - REFRESH_TOKEN_SECRET=dbb85xxxxxx9ebde
+    ports:
+      - 4001:4001
```

Comment:
For `network_mode: host`, you can't use port bindings. This section could be replaced with:
```yaml
expose:
  - 4001
```